.github/workflows/ci_sdk.yml

```diff
@@ -52,4 +52,4 @@ jobs:
         run: curl http://0.0.0.0:8000/inspect-name/mainnet/test.eth
 
       - name: Run tests
-        run: NAMEGUARD_TEST_ENDPOINT=http://0.0.0.0:8000 pnpm test -r
+        run: NAMEGUARD_API_URI=http://0.0.0.0:8000 pnpm test -r
```

Comment:
Am I correct to understand that this is passing in `NAMEGUARD_API_URI` as an environment variable to `pnpm test -r`?

If so, what do you think about updating our syntax here so that we pass in the environment variable similar to how we do it here: https://github.com/namehash/namekit/blob/main/.github/workflows/ci_sdk.yml#L56-L57

Reply:
Yes.